src/components/LoginDialog/LoginDialog.vue

```diff
@@ -0,0 +1,104 @@
+<template>
```

Comment:
I wasn't sure how to call it, since it isn't always a "login dialog", ideas are welcome.

Reply:
Maybe we can create an "Authentication component" that has two separate components: login and logout?